uber/automated_emails.py

```diff
@@ -22,13 +22,18 @@
          needs_approval=False, allow_during_con=True,
          ident='attendee_payment_received')
 
+AutomatedEmail(Attendee, '{EVENT_NAME} registration confirmed', 'reg_workflow/attendee_confirmation.html',
+                lambda a: a.paid == c.NEED_NOT_PAY and not a.placeholder,
+                needs_approval=False, allow_during_con=True,
+                ident='attendee_badge_confirmed')
+
 AutomatedEmail(Group, '{EVENT_NAME} group payment received', 'reg_workflow/group_confirmation.html',
          lambda g: g.amount_paid == g.cost and g.cost != 0,
          needs_approval=False,
          ident='group_payment_received')
 
 AutomatedEmail(Attendee, '{EVENT_NAME} group registration confirmed', 'reg_workflow/attendee_confirmation.html',
-         lambda a: a.group and a != a.group.leader and not a.placeholder,
+         lambda a: a.group and (a != a.group.leader or a.group.cost == 0) and not a.placeholder,
```

Comment:
You'll see above that we have a "group payment received" email that goes to the group (aka the group leader), but only if the group has a cost. Meanwhile, this email, which goes to persons who claim badges from a group, would exclude the group leader. The end result is that if an admin makes a free group, its leader never gets a confirmation email besides possibly the placeholder email. I figured since I was fixing other emails, I should add a fix for this.